docs/Policy/Maintenance/Scenario_Maintenance_Policy.md

```diff
@@ -0,0 +1,189 @@
+- [Scenario Expansions](#scenario-expansions)
+  - [Updating the Scenario Config](#updating-the-scenario-config)
+- [Scenario Deprecations](#scenario-deprecations)
+- [OpenShift Deployment](#openshift-deployment)
+- [Layered Product Deployment](#layered-product-deployment)
+- [Test Execution](#test-execution)
+- [Triggering Cadence](#triggering-cadence)
+- [Reporting Tests](#reporting-tests)
+- [Skipping Scenarios](#skipping-scenarios)
+
+## Overview
+You'll see in this [RACI Chart](../../Onboarding/RACI_Chart.md) that there is a shared responsibility for the maintenance of these openshift CI scenarios. This document is meant to describe the tasks included in maintaining a scenario and state who should be responsible.
+
+We define scenario maintenance as:
+> Any task needed to ensure that the scenario executes correctly when automatically triggered.
+
+Scenario maintenance tasks include:
+1. Scenario Expansions (PQE)
+2. Scenario Deprecations (PQE)
+3. OpenShift Deployment (Workflow OWNERS)
+4. Layered Product Deployment (PQE)
+5. Test Execution (PQE)
+6. Triggering Cadence (CSPI-QE)
+7. Reporting Tests (CSPI-QE)
+8. Skipping Scenarios (CSPI-QE)
+
+## Scenario Expansions
+> This is the responsibility of the PQE team.
+
+A scenario expansion can be defined as preparing your test scenario to execute tests against a newer layered product release then was previously configured. For example, if the scenario executes tests for `ACM2.6 on OCP4.12` then an expansion would mean to extend the config for this scenario to now include testing for `ACM2.7 on OCP4.12`
+
+### Updating the Scenario Config
+Here we have the `tests"` section of a scenario config for the ACM scenario.
+
+```
+tests:
+- as: acm-interop-aws
+  cron: 0 1 * * 1
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: aws.interop.ccitredhat.com
+      SUB_CHANNEL: release-2.6
+      SUB_INSTALL_NAMESPACE: open-cluster-management
+      SUB_PACKAGE: advanced-cluster-management
+      SUB_SOURCE: redhat-operators
+      SUB_TARGET_NAMESPACES: open-cluster-management
+    test:
+    - chain: interop-acm
+    workflow: ipi-aws
+```
+
+This is showing us that the layered product being installed is being controlled by the `SUB_CHANNEL` env variable. This will install the `ACM2.6` version of the operator from the operatorhub.
+
+If our goal is to expand the scenario so that we also test `ACM2.7` then we just need to create a new test stanza with an updated value for `SUB_CHANNEL`. So now we will have.
+
+```
+tests:
+- as: acm-interop-aws
+  cron: 0 1 * * 1
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: aws.interop.ccitredhat.com
+      SUB_CHANNEL: release-2.6
+      SUB_INSTALL_NAMESPACE: open-cluster-management
+      SUB_PACKAGE: advanced-cluster-management
+      SUB_SOURCE: redhat-operators
+      SUB_TARGET_NAMESPACES: open-cluster-management
+    test:
+    - chain: interop-acm
+    workflow: ipi-aws
+- as: acm-interop-aws
```

Comment:
Have you tested having two tests with the same `as` value? If this works, just disregard this comment. Otherwise, we may need to do `acm-interop-aws-2.6` and `acm-interop-aws-2.7`. We may want to do that anyway when we are reporting to testgrid.

Reply:
Good catch I have tested this and it does not work, I just failed to update this.

fixing in next commit.